src/main/java/org/zalando/stups/tokens/fs/AccessTokenDto.java

```diff
@@ -21,8 +21,9 @@
 
     private final String name;
 
-    AccessTokenDto(String secret, String type, String name) {
-        super(secret, type, -1, null);
+    public AccessTokenDto(AccessToken accessToken, String name) {
```

Comment:
can we leave this package private? This will make future refactorings easier, because we don't need to care about changing public interfaces ;-)